### ISSUE TITLE
Python process is buffering stdout

### ISSUE THREAD

**Issue by queued37** [queued37/codejam-template#1]
This can be solved by redefining `print()` as
```python
def print(*args, **kwargs):
    __builtins__.print(*args, **kwargs)
    sys.stdout.flush()
```
or, equivalently:
```python
def print(*args, **kwargs):
    __builtins__.print(*args, **kwargs, flush=True)
```

**Comment by queued37** [comment]
Better solution: use `-u` option.
```javascript
const pythonProcess = spawn('python', ['-u'].concat(argv._));
```

**Comment by queued37** [comment]
Fixed by a0e1e3b
